src/Symfony/Component/Serializer/Normalizer/CacheableSupport.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Normalizer;
+
+/**
```

Comment:
The codestyle claims this should be a single-asterisk docblock (`/*`), even though classes and interfaces do have `/**`, is the check perhaps not yet updated to support enums?

```suggestion
/*
```

Reply:
imho it's a bug in fabbot's config